qt/scientific_interfaces/Indirect/test/FqFitModelTest.h

```diff
@@ -91,13 +93,32 @@ class FqFitModelTest : public CxxTest::TestSuite {
         "f1.FWHM");
   }
 
+  void test_that_zeroWidths_returns_true_if_the_workspace_contains_no_widths() {
```

Comment:
Why can this not be the same format as its EISF equivilant?

Reply:
I tried that (see commit 2467fe9) but the test failed. I went back to the original code and copied the original test that tested getWidths but removed the zeroWidths assertion as that method no longer exists.

Reply:
Ok so long as it is not a reversion